### PR TITLE
Press to eloquent

### PR DESCRIPTION
**Pull request by smy0906**
remove IntraDb dependency and duplicate codes.

### CODE REVIEW

File: tests/PressTest.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    }
+
+    public function testRemovePress() {
+        $this->assertTrue(false);
+    }
+
+    public function testGetCount(){
+        $this->assertTrue(false);
+    }
+}
```
Comment (namenu) [inline tests/PressTest.php]: I see no EOL here.
After merge master, running 'php-cs-fixer fix' will fix it automatically.

File: src/Intra/Service/Press/Press.php [hunk 2]
```diff
@@ -70,54 +57,63 @@ public function del($press_id)
...
+
+        // if it works even there is needless 'count', it can be one. I am not sure.
+        if (is_null($count)) {
+            $json_dto->data = [
+                'user' => $this->user,
+                'press' => $press,
+                'manager' => UserSession::isPressManager()
+            ];
+        } else {
+            $json_dto->data = [
```
Comment (namenu) [inline src/Intra/Service/Press/Press.php]: These assignments are duplicated.
Consider adding 'count' key conditionally when count is not null.

File: src/Intra/Service/Press/Press.php [hunk 3]
```diff
@@ -70,54 +57,63 @@ public function del($press_id)
...
     }
 
     private function getPressCount()
     {
-        $db = IntraDb::getGnfDb();
+        return PressModel::count();
+    }
 
-        return $db->sqlData('select count(*) from press');
+    private function makeRespone($press, $count = null) {
```
Comment (namenu) [inline src/Intra/Service/Press/Press.php]: Found typo for response.
Also, to make it more clear, how about 'makeJsonResponse' ?

Comment (smy0906) [inline src/Intra/Service/Press/Press.php]: I agree with it.

**Comment by smy0906** [comment]
I fixed them from your feedback.

**Comment by namenu** [comment]
:+1:

**Pull request merged**